src/interface/storage.thrift

```diff
@@ -203,6 +205,40 @@ struct CatchUpDataReq {
     3: common.HostAddr     target,
 }
 
+struct UpdateResponse {
+    1: required ResponseCommon result,
+    2: optional common.Schema schema,   // return column related props schema
+    3: optional binary data,            // return column related props value
+    4: optional bool upsert = false,    // it's true when need to be inserted by UPSERT
+}
+
+struct UpdateItem {
+    1: required binary name,    // TagName or EdgeType
+    2: required binary field,   // property name
+    3: required binary value,   // expression which is encoded
+}
+
+struct UpdateVertexRequest {
+    1: common.GraphSpaceID space_id,
+    2: common.VertexID vertex_id,
+    3: common.PartitionID part_id,
+    4: binary filter,
+    5: list<UpdateItem> update_items,
+    6: list<PropDef> return_columns,
+    7: bool insertable,
+    8: common.EdgeType edge_type,
```

Comment:
Why we need edge_type in UpdateVertex request?

Reply:
In storage, both UpdateVertexProcessor and UpdateEdgeProcessor call the function QueryBaseProcessor::checkAndBuildContexts first, in their own checkAndBuildContexts functions.  The QueryBaseProcessor::checkAndBuildContexts  should use the req.edge_type.

FYI. https://github.com/vesoft-inc/nebula/blob/master/src/storage/QueryBaseProcessor.inl#L39

Reply:
Don't design your interface as implementation internal.

Reply:
It has been refactored which remove edge_type away.